Evaluator/TA/volatility_evaluator.py

```diff
@@ -0,0 +1,55 @@
+"""
+OctoBot Tentacle
+
+$tentacle_description: {
+    "package_name": "OctoBot-Tentacles",
+    "name": "volatility_evaluator",
+    "type": "Evaluator",
+    "subtype": "TA",
+    "version": "1.1.0",
+    "config_files": ["StochasticRSIVolatilityEvaluator.json"],
+    "requirements": []
+}
+"""
+
+#  Drakkar-Software OctoBot
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import tulipy
+
+from config import PriceIndexes, START_PENDING_EVAL_NOTE
+from evaluator.TA.TA_evaluator import VolatilityEvaluator
+
+
+class StochasticRSIVolatilityEvaluator(VolatilityEvaluator):
```

Comment:
Can you add a description to be displayed in web interface under DESCRIPTION classparam ?
ex:
```python
class RSIMomentumEvaluator(MomentumEvaluator):
    DESCRIPTION = "Uses the Relative Strength Index (length of 14) to find trend reversals. When found, evaluates -1 to 1" \
                  " according to the strength of the RSI."
```

Reply:
Yes !